### DISCUSSION TITLE
Add border style when validation failed for TailwindCSS?

### DISCUSSION THREAD

**Discussion by daFish** [orchidjs/tom-select#540] [Q&A]
Hi there,

I'm trying to add a custom border style when a validation failed like in Bootstrap: 
![Bildschirm­foto 2022-12-23 um 10 16 35](https://user-images.githubusercontent.com/150416/209309202-39b3851d-7299-4f78-81df-2b2de9771dbd.png)

Added my custom classes doesn't work so I guess I miss something. Any help or hint is greatly appreciated.

**Comment by bucari** [comment]
Are you using tom-select with TailwindCSS? 

Can you share your css?

**Comment by daFish** [comment]
This can be achieved by applying custom styles:

```
.ts-wrapper.is-invalid,
.was-validated .invalid,
.was-validated :invalid + .ts-wrapper {
    @apply border-red-500 text-red-500;
}

.ts-wrapper.is-invalid .ts-control,
.was-validated .invalid.focus .ts-control,
.was-validated :invalid + .ts-wrapper.focus .ts-control {
    @apply border-red-500 text-red-500;
}
```

Make sure the element has the class `is-invalid`.
